seokhwan/1/문자열_내_p와_y의_개수.js

```diff
@@ -0,0 +1,12 @@
+function solution(s) {
+  let pCount = 0;
+  let sCount = 0;
+
+  for (let i = 0; i < s.length; i++) {
+    if (s[i] === "p" || s[i] === "P") pCount++;
+    else if (s[i] === "y" || s[i] === "Y") sCount++;
+  }
+
+  const answer = pCount === sCount ? true : false;
+  return answer;
```

Comment:
비교값을 const로 선언한 변수에 할당하고 변수를 return 하는 방법도 좋지만
```suggestion
  return pCount === sCount ? true : false;
```
이런 방법도 있다고 말씀드리고 싶네요~~